### PR TITLE
Make mongodb backup one database at a time

### PR DESCRIPTION
**Pull request by xitij2000**
This pull request makes MongoDB backup databases one at a time instead of all at once and inserts a small delay between backups.

### CODE REVIEW

File: templates/backup-pre-mongodb.sh [hunk 1]
```diff
@@ -4,4 +4,11 @@ TMPDIR='{{ MONGODB_SERVER_BACKUP_DIR }}'
 
 rm -rf "$TMPDIR"
 sudo -u mongodb mkdir -p "$TMPDIR"
-sudo -u mongodb mongodump -u '{{ mongodb_root_admin_name }}' -p '{{ mongodb_root_admin_password }}' -o "$TMPDIR"
+DBNAMES=`echo "show databases;" | \
+         sudo -u mongodb mongo --quiet -u '{{ mongodb_root_admin_name }}' -p '{{ mongodb_root_admin_password }}' 127.0.0.1:{{ mongodb_net_port }}/admin | \
+         awk '{ print $1; }'`
+
+for DBNAME in $DBNAMES; do
+    sudo -u mongodb mongodump -u '{{ mongodb_root_admin_name }}' -p '{{ mongodb_root_admin_password }}' --authenticationDatabase admin --db '$DBNAME' --out "$TMPDIR"
```
Comment (clemente) [inline templates/backup-pre-mongodb.sh]: `'$DBNAME'` should be `"$DBNAME"` to do interpolation.

**Review by clemente — COMMENTED** [review]
Small change to do. But wait, I'll comment on the task.

File: templates/backup-pre-mongodb.sh [hunk 2]
```diff
@@ -4,4 +4,11 @@ TMPDIR='{{ MONGODB_SERVER_BACKUP_DIR }}'
...
+    sleep 5
```
Comment (clemente) [inline templates/backup-pre-mongodb.sh]: (And without the sleep it will be much faster. And it works)

**Comment by xitij2000** [comment]
Closed because this approach doesn't bring any performance benefit.

**Pull request closed without merging**